annotation-server/src/medications/medications.service.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { ConfigModule } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+
+import { Medication } from './medication.entity';
+import { MedicationsService } from './medications.service';
+
+describe('MedicationsService', () => {
+    let medicationsService: MedicationsService;
+
+    beforeEach(async () => {
+        const modelFixture: TestingModule = await Test.createTestingModule({
+            imports: [ConfigModule.forRoot()], // needed for MedicationService
```

Comment:
This uses the `.env` instead of the `test/.env`, right? Do we want it that way?

Reply:
yes, that's right. The ConfigModule is only needed in the background and not explicitly used to read environment variables in the test though. That's why I didn't think the ``.env``-path mattered. Is it a better practice to change it anyway?

Reply:
I think changing it to the testing env would be good; someone might add a test later and expect this.

Reply:
What do you mean by "not explicitly used to read environment variables in the test"?
I just tested it with the following code:
```javascript
ConfigModule.forRoot({
    envFilePath: ['test/.env', '.env'],
})
```
Then it used the test-environment over the normal env-variables (like in the normal `app.module.ts`), which leads to different behaviour. Without it the test `MedicationsService › getDataFromJSON › should return right drugs from example ZIP` fails, because my normal environment contains the example data with 100 medications and not 2 medications like the file linked in the test env.

Reply:
as far as I can tell, both 'test/.env' and '.env' contain the same xml and zip paths (see [here](https://github.com/hpi-dhc/PharMe/pull/226/files#diff-5e9ab17d245f72d9cb8b748d1246eab3d9da7c1f3d35d12751fa0ef6cb375ab1), so this should not be happening I think. 
It does not really matter though, I will change it :)

Reply:
That it definitely true for the example `.env`, but for example I changed the name of the normal zip to represent the drug bank data instead of the example data. As soon as this happens, the differentiation is needed.

Reply:
good point, thank you!